### DISCUSSION TITLE
Explicit content warner

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#16905] [Ideas] [answered]
Maybe add feature which part of beatmap is explicit and so it can warn user to say that  
*This beatmap has a \<content\> element which is marked as explicit. Do you want to continue?*  
And there should be option for *Don't warn me if beatmap have any explicit element*  
Very cool for playing osu outside to prevent anyone hear immature word or boobas in background images

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/11460
